src/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,39 @@
+import React = require('react');
+
+export interface CheckBoxProps {
```

Comment:
from the typescript language spec: https://github.com/Microsoft/TypeScript/blob/master/doc/spec.md#3.10

"Interface types have many similarities to type aliases for object type literals, but since interface types offer more capabilities they are generally preferred to type aliases"

@tobich ?